src/test_l1_dsCompositeIn.c

```diff
@@ -292,7 +292,7 @@ void test_l1_dsCompositeIn_positive_dsCompositeInGetNumberOfInputs(void)
     DS_ASSERT_AUTO_TERM_NUMERICAL(dsCompositeInGetNumberOfInputs(&numberOfInputs1), dsERR_NONE);
 
     // Step 03: Get number of inputs for the second time
-    DS_ASSERT_AUTO_TERM_NUMERICAL(dsCompositeInGetNumberOfInputs(&numberOfInputs2), dsERR_NONE);
+    numberOfInputs2=ut_kvp_getUInt16Field(ut_kvp_profile_getInstance(), "composite_input_configurations/number_of_ports");
```

Comment:
Please update to the latest composite Input profile, path should be "dsCompositeIn/composite_input_configurations/number_of_ports"

Reply:
done